test/integration/dnspolicy_controller_test.go

```diff
@@ -882,343 +1199,4 @@ var _ = Describe("DNSPolicy", Ordered, func() {
-					return fmt.Errorf("expected %v probes, got %v", 2, len(probeList.Items))
-				}
-				return nil
-			}, TestTimeoutMedium, TestRetryIntervalMedium).Should(BeNil())
-			Expect(len(probeList.Items)).To(Equal(2))
-		})
-
-		Context("all unhealthy probes", func() {
-			It("should publish all dns records endpoints", func() {
-				lbHash = dns.ToBase36hash(fmt.Sprintf("%s-%s", gateway.Name, gateway.Namespace))
-
-				expectedEndpoints := []*v1alpha1.Endpoint{
-					{
-						DNSName: "2w705o.lb-" + lbHash + ".test.example.com",
-						Targets: []string{
-							TestAttachedRouteAddressTwo,
-						},
-						RecordType:    "A",
-						SetIdentifier: "",
-						RecordTTL:     60,
-					},
-					{
-						DNSName: "s07c46.lb-" + lbHash + ".test.example.com",
-						Targets: []string{
-							TestAttachedRouteAddressOne,
-						},
-						RecordType:    "A",
-						SetIdentifier: "",
-						RecordTTL:     60,
-					},
-					{
-						DNSName: "default.lb-" + lbHash + ".test.example.com",
-						Targets: []string{
-							"2w705o.lb-" + lbHash + ".test.example.com",
-						},
-						RecordType:    "CNAME",
-						SetIdentifier: "2w705o.lb-" + lbHash + ".test.example.com",
-						RecordTTL:     60,
-						ProviderSpecific: v1alpha1.ProviderSpecific{
-							{
-								Name:  "weight",
-								Value: "120",
-							},
-						},
-					},
-					{
-						DNSName: "default.lb-" + lbHash + ".test.example.com",
-						Targets: []string{
-							"s07c46.lb-" + lbHash + ".test.example.com",
-						},
-						RecordType:    "CNAME",
-						SetIdentifier: "s07c46.lb-" + lbHash + ".test.example.com",
-						RecordTTL:     60,
-						Labels:        nil,
-						ProviderSpecific: v1alpha1.ProviderSpecific{
-							{
-								Name:  "weight",
-								Value: "120",
-							},
-						},
-					},
-					{
-						DNSName: "lb-" + lbHash + ".test.example.com",
-						Targets: []string{
-							"default.lb-" + lbHash + ".test.example.com",
-						},
-						RecordType:    "CNAME",
-						SetIdentifier: "default",
-						RecordTTL:     300,
-						ProviderSpecific: v1alpha1.ProviderSpecific{
-							{
-								Name:  "geo-code",
-								Value: "*",
-							},
-						},
-					},
-					{
-						DNSName: "test.example.com",
-						Targets: []string{
-							"lb-" + lbHash + ".test.example.com",
-						},
-						RecordType:    "CNAME",
-						SetIdentifier: "",
-						RecordTTL:     300,
-					},
-				}
-
-				err := k8sClient.Get(ctx, client.ObjectKey{Name: gateway.Name, Namespace: gateway.Namespace}, gateway)
-				Expect(err).NotTo(HaveOccurred())
-				patch := client.MergeFrom(gateway.DeepCopy())
-				addressType := mgcgateway.MultiClusterIPAddressType
-				gateway.Status.Addresses = []gatewayv1beta1.GatewayAddress{
-					{
-						Type:  &addressType,
-						Value: fmt.Sprintf("%s/%s", "kind-mgc-control-plane", TestAttachedRouteAddressOne),
-					},
-					{
-						Type:  &addressType,
-						Value: fmt.Sprintf("%s/%s", "kind-mgc-control-plane", TestAttachedRouteAddressTwo),
```

Comment:
I notice this is removed from the test where it's copied to. Is the expectation that the status.address update is covered now by the reconciler or by some change to the test configuration?

Reply:
Right now it is moved [here](https://github.com/Kuadrant/multicluster-gateway-controller/pull/465/files#diff-2adf2dbe2f36d347bda43e252bea13a9d720c5e9548c14ef693d41433d9da4c6R96-R107), but we  do hope to have it [done](https://github.com/Kuadrant/multicluster-gateway-controller/issues/330) by the controller at some point